scripts/getArchive.py

```diff
@@ -51,6 +51,14 @@ def matches_filters(ent, args):
                 return False
         return True
```

Comment:
You're duplicating this code for the case where `ent` is either a dict or...I'm not sure what. Can you convert one of them so that this function doesn't need to be duplicated?

Reply:
The `json` library parses the feed into a dict, and the protobuf library parses the feed into an object-oriented structure (`gtfs_realtime_pb2.FeedMessage`). The underlying data has the same format, but requires different syntax to retrieve it.

Json would be easier output to work with than the almost-json that is output by the protobuf parser, but I'm not sure how I would programatically convert the object nesting and class variables into the same nested dictionary structure as the json.

Reply:
https://github.com/benhodgson/protobuf-to-dict might work?

Reply:
looks like it's not updated for python3. PR submitted last january but no activity on it since it was opened

Reply:
https://github.com/JeffreyWardman/protobuf-to-dict might do it?

Reply:
Is there a way to use that fork (instead of the published version on pypi) in a way that's easy to describe in the readme?